packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -603,6 +603,14 @@ export function getJobStatisticsHtml(jobStatistics: any) {
     return htmlString
 }
 
+export function getBillingString(linesOfCode: number) {
```

Comment:
See above comment, you shouldn't need to manually replace magic strings in your text. String interpolation can do this for you

Reply:
See my reply on other PR; not opposed to String interpolation but then my String constants will get disorganized.

Reply:
Edit: fixed